presto-docs/src/main/sphinx/release/release-0.215.rst

```diff
@@ -0,0 +1,53 @@
+=============
+Release 0.215
+=============
+
+General Changes
+---------------
+
+* Add dynamic schedule support for grouped execution . When a stage can be executed
+  with grouped execution and the stage has no remote source, table partitions can be
+  scheduled to tasks in a dynamic way. This can mitigate skew for grouped executed queries.
+  The dynamic schedule feature for grouped execution must be enabled with the
+  ``dynamic_schedule_for_grouped_execution`` config property or the
+  ``dynamic-schedule-for-grouped-execution`` config property.
+* Enable spatial joins for join conditions expressed with :func:`ST_Within`
+* Improve spatial join performance by accelerating build-side geometries.
+* Add input validation checks to :func:`ST_LineString` to conform with the specification
```

Comment:
Need a "." at the end. Would be nice if we specify which spec we are talking about. /cc @mbasmanova